backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/PluginsSynchronizer.java

```diff
@@ -61,8 +61,13 @@ public PluginSynchronizationSummary synchronize(ServerApi serverApi, boolean sup
     var pluginsToDownload = serverPlugins.stream()
       .filter(p -> shouldDownload(p, storedPluginsByKey))
       .collect(Collectors.toList());
+
+    if(pluginsToDownload.isEmpty()) {
```

Comment:
Same here. Is it a misconfiguration of the formatter?

Reply:
I fixed the config, thanks